docs/rest-api.md

```diff
@@ -2,7 +2,9 @@
 
 > **NOTE**
 >
-> Azure SignalR Service only supports using REST API to manage clients connected using ASP.NET Core SignalR. Clients connected using ASP.NET SignalR use a different data protocol and is now not supported.
+> * Azure SignalR Service only supports using REST API to manage clients connected using ASP.NET Core SignalR. Clients connected using ASP.NET SignalR use a different data protocol and is now not supported.
+> 
```

Comment:
How about simplify a little like: Azure SignalR Service REST API only supports managing clients using **ASP.NET Core SignalR**. Clients connected using ASP.NET SignalR is not supported now.